README.md

```diff
@@ -6,17 +6,17 @@
 
 This jenkins plugin allow autodiscovery of Tuleap's git repositories and branches to automatically create jenkins jobs when branches have `Jenkinsfile`.
 
-Works with any recent version of Jenkins >= 2 (latest LTS preferred).
+Works with any recent version of Jenkins >= 2.150.3 (latest LTS preferred).
 
 # How to use
 
-* Install the plugin (!)
-* In Jenkins global configuration you should reference your Tuleap instance (only one server ATM)
+* Install the plugin
+* In Jenkins global configuration you should reference your Tuleap instance (There is a limitation of 1 Tuleap server per Jenkins instance)
 * Then any Jenkins user can create a new "Tuleap project" job type
-  * They should have a `username/password` jenkins credential with a user that can access the project you want to target
+  * They should have a `Tuleap Access Key` jenkins credential with a user that can access the project you want to target
```

Comment:
I think you can link Tuleap Access Key with the dedicated doc: https://docs.tuleap.org/user-guide/integration/rest/quick-start/auth.html

What do you think ?

Reply:
Even better https://docs.tuleap.org/user-guide/citizen.html#access-keys